project/src/test/java/com/google/sps/data/PlacesTest.java

```diff
@@ -114,7 +97,7 @@ public void filter_filterNotRequired_notFiltering() {
     ImmutableList<Place> allPlaces = ImmutableList.of(websiteEmpty, samePlace1, samePlace2);
 
     ImmutableList<Place> result = Places.filter(
-      allPlaces /* places */,
+        allPlaces /* places */,
```

Comment:
Usually if there's a name to the variable, you don't need to explain what is this variable (i.e. remove the `/* places */ ` comment), it only adds unnecessary explanations to the reader.

Reply:
Great point, fixed